dto/decision_dto.go

```diff
@@ -80,6 +82,7 @@ func NewAPIDecision(decision models.Decision) APIDecision {
 		Score:                decision.Score,
 		Rules:                make([]APIDecisionRule, len(decision.RuleExecutions)),
 		ScheduledExecutionId: decision.ScheduledExecutionId,
+		ScenarioIterationId:  decision.ScenarioIterationId,
```

Comment:
Just in case: there is `Version` info in the `Scenario` property. It may be better to flatten everything or group everything or 🤷🏽 

I do not have strong opinion but I think grouping `Version` + `ScenarioIterationId` at the same place (or droping `Version` make sense)

Having a copy of Scenario `Name` + `Description` can make sense since they are mutable

Reply:
ah yes, of course 👍